### PR TITLE
Use remembered credential even if it is empty

### LINKED ISSUE #875: [Bug] Remember ssh key passphrase
App Version: 1.9

Since version 1.9 the "Remember passphrase" feature for the ssh key is not working anymore.

After pulling down two or three times to synchronise the remote git repo I get asked for the ssh passphrase.

### PR DESCRIPTION
**Pull request by fmeum**
## :loudspeaker: Type of change

- [x] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

## :scroll: Description

Use the remembered credential even if it is empty.

## :bulb: Motivation and Context

Should fix #875.

## :green_heart: How did you test it?
Can't test right now, but am quite positive that this is the fix.

## :pencil: Checklist

- [x] I formatted the code with the IDE's reformat action (Ctrl + Shift + L/Cmd + Shift + L)
- [x] I reviewed submitted code
- [x] I added a [CHANGELOG](CHANGELOG.md) entry if applicable

### CODE REVIEW

**Comment by fmeum** [comment]
I confirmed that this fixes the issue. Should we revert https://github.com/android-password-store/Android-Password-Store/commit/e1f81e8f4b56246706564680f4ab5d6d10f2561e before releasing 1.9.1?

**Comment by msfjarvis** [comment]
I was gonna pitch a new branching and versioning system anyway so might as well. How about we split off a new branch from v1.9.0 called release, rename master to develop and then backport patches from develop to the release branch for our patch releases? Tags will continue pointing to the right commits so merging release back into develop won't be a problem.

**Comment by fmeum** [comment]
Sounds good. I won't be able to do this today and maybe not even tomorrow, but I would have proposed something very similar.

**Comment by msfjarvis** [comment]
> Sounds good. I won't be able to do this today and maybe not even tomorrow, but I would have proposed something very similar.

Yeah no problem, I'll do it in a couple hours.

**Comment by msfjarvis** [comment]
Okay I unfucked the PR situation and I think we're good to go now.

**Comment by msfjarvis** [comment]
Generated another key with `-m PEM`, gave it an empty passphrase and wasn't prompted on subsequent operations so I guess we're good to go.

**Review by msfjarvis — DISMISSED** [review]

**Review by msfjarvis — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Fixed
+
+-   Remember passphrase option did not work with keys that had empty passphrases and were generated before 2018
```
Comment (fmeum) [inline CHANGELOG.md]: Make it 2019 to be safe ;-)

**Review by msfjarvis — APPROVED** [review]

**Pull request merged**